cookbook/integrations/aws/athena/athena.py

```diff
@@ -0,0 +1,84 @@
+from flytekit import dynamic, kwtypes, task, workflow
+from flytekit.types.file import FlyteFile
+from flytekit.types.schema import FlyteSchema
+from flytekitplugins.athena import AthenaConfig, AthenaTask
+
+# %%
+# This is the world's simplest query. Note that in order for registration to work properly, you'll need to give your
+# Athena task a name that's unique across your project/domain for your Flyte installation.
+athena_task_no_io = AthenaTask(
+    name="sql.athena.no_io",
+    inputs={},
+    query_template="""
+        select 1
+    """,
+    output_schema_type=None,
+    task_config=AthenaConfig(database="mnist"),
+)
+
+
+@workflow
+def no_io_wf():
+    return athena_task_no_io()
+
+
+# %%
+# Of course, in real world applications we are usually more interested in using Athena to query a dataset.
+# In this case we assume that the vaccinations table exists and has been populated data according to this schema
```

Comment:
This is where I got the vaccination dataset from: https://www.kaggle.com/gpreda/covid-world-vaccination-progress

maybe we should link to that here?

Reply:
ah perfect, thank you!

Reply:
```suggestion
# In this case we assume that the vaccinations table exists and has been populated data according to this schema:
```

Reply:
done